src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts

```diff
@@ -17,7 +17,10 @@ describe('DotEditLayoutService', () => {
                 identifier: '5363c6c6-5ba0-4946-b7af-cf875188ac2e',
```

Comment:
migrate to `TestBed`

Reply:
done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-9fdb7f7a5d8c1c883513d66fe700c574R93